skate-plugin/src/main/kotlin/com/slack/sgp/intellij/util/StringExtensions.kt

```diff
@@ -20,3 +20,11 @@ import com.google.common.base.CaseFormat
 fun String.snakeToCamelCase(): String {
   return CaseFormat.LOWER_UNDERSCORE.to(CaseFormat.LOWER_CAMEL, this)
 }
+
+fun String.getJavaPackageName(): String? {
```

Comment:
This feels tricky to be putting on string. Presumably we usually have a File or Path, could we use that and then get a concrete list of parent dirs and relativize them to the original `src` source set?

Reply:
Moved this to `PathsExtension` here https://github.com/slackhq/slack-gradle-plugin/pull/799/commits/6986e693d5d0bf42d69c263050612fb0634dbdbc#diff-2ce36a3822d274d7a4642aba1324108f9e61db2fe992d7f11aabf443147cce97R20